src/Workspaces/Core/Portable/Shared/Extensions/ITypeSymbolExtensions.cs

```diff
@@ -894,6 +895,34 @@ public static bool IsEnumType(this ITypeSymbol type)
                 type = type.GetTypeArguments()[0];
             }
 
+            switch (type.SpecialType)
```

Comment:
Is it possible to add a unit test?

Reply:
Not in any straightforward manner. We will be able to once https://github.com/dotnet/roslyn-sdk/pull/329 is available.

Reply:
The behavior anomaly is an artifact of the *reference assembly* in [NETStandard.Library 2.0.0](https://www.nuget.org/packages/NETStandard.Library/2.0.0) and does not occur with the implementation or runtime assemblies. It also doesn't occur with reference assemblies produced by the current C# compiler because the underlying concern was already addressed.